js/src/stats.js

```diff
@@ -95,6 +95,26 @@ module.exports = (common) => {
       })
     })
 
+    it('.bw Poll', (done) => {
+      if (!withGo) {
+        console.log('Not supported in js-ipfs yet')
+        return done()
+      }
+
+      ipfs.stats.bw({poll: true}, (err, res) => {
+        expect(err).to.not.exist()
+        expect(res).to.exist()
+
+        res.once('data', (data) => {
+          expect(data).to.have.a.property('totalIn')
```

Comment:
This expectation only checks that the property exists, which I don't think is good enough. These test cases would still pass even if `totalIn` is set to for example `false`, `undefined` or even `null`, which is not correct.

Reply:
Noticed that the two test cases are the same as well. One should test with `{poll: true}` and the other one without passing any extra args.

Reply:
We have 4 test cases: 2 for callbacks and two for promises. One of those two has `{poll: true}` and the other has no args.

Right now, those values are strings. Should I convert them to something like [`big.js`](https://github.com/MikeMcl/big.js/)?

Reply:
Added a `try catch` to check if they are numbers using `big.js.`